src/client/ClientDataResolver.js

```diff
@@ -360,6 +361,7 @@ class ClientDataResolver {
    */
   static resolveColor(color) {
     if (typeof color === 'string') {
+      if (color === 'RANDOM') return Math.floor(Math.random() * 16777214) + 1;
```

Comment:
"Uhh, technically black isn't a colour - it's the lack of colour, you stupid dimwit."
- Gus